hubmap/query_app/serializers.py

```diff
@@ -223,6 +223,14 @@ class Meta:
             "clusters",
         ]
 
+    def get_clusters(self, obj):
+        clusters_list = []
+        for cluster in obj.clusters.all():
```

Comment:
Unless I'm missing something (like forgetting whether a field is dynamically generated on the model class instead of being stored in the database), this seems like it could be more efficient with something like `obj.clusters.values_list('grouping_name', flat=True)`.